prisma/schema.prisma

```diff
@@ -68,8 +68,8 @@ model Profile {
   bio         String?
   githubUrl   String?
   profilePicture String?
-  specialism  String? @default("")
-  mobile      String? @default("")
```

Comment:
Why are we dropping the default?

Reply:
To clean up the returned data. Looked a bit odd to only have mobile and specialism set to "" when several other variables in the same table are returning null.

Reply:
Ok